### PR TITLE
Javascript javascript1 week1

### PR DESCRIPTION
**Pull request by rebeccahockley**
Finished Week 1 Homework

### CODE REVIEW

File: javascript/javascript1/week1/Homework.js [hunk 1]
```diff
@@ -1,5 +1,5 @@
 console.log("Age-ify");
 let yearOfBirth = 1997;
 let yearFuture = 2121;
-const age = yearFuture - yearOfBirth;
+let age = yearFuture - yearOfBirth;
 console.log("You will be " + age + " years old in " + yearFuture);
```
Comment (MLMeyer) [inline javascript/javascript1/week1/Homework.js]: Good to see you reflecting on whether the variables should be const or let. Which ones could be const and why? 😊

File: javascript/javascript1/week1/dogcalculator.js [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+if (shouldShowResultInDogYears === true) {
+  console.log(
+    "Your dog will be " + dogAge + " years old in " + dogYearFuture + "."
+  );
+} else shouldShowResultInDogYears === false;
+{
+  console.log(
+    "Your dog will be " + dogYear + " human years old in " + dogYearFuture + "."
+  );
+}
```
Comment (MLMeyer) [inline javascript/javascript1/week1/dogcalculator.js:24]: Good work. You have a good structure and some nice logic. 😊

**I have a few observations and suggestions 💫**
- Could the variable dogYear have a different name that represents the value even clearer? 
- Could one of the variables be a constant? 

Tip 💡
In the if else statement you typically define the condition in the if clause - as you have done very nicely. If this condition is false we will automatically move on to the else clause, and here you do not specify a condition. Maybe I can visualize it through pseudocode below. 

if (condition) {
  //  block of code to be executed if the condition is true
} else {
  //  block of code to be executed if the condition is false
}

Here is another source, that might explain it better than me 😇😊
https://www.w3schools.com/js/js_if_else.asp

File: javascript/javascript1/week1/houseprice.js [hunk 3]
```diff
@@ -0,0 +1,43 @@
...
+      " Julia is paying too much!"
+  );
+} else {
+  console.log(
+    "The cost of Julia's house is " +
+      housePriceJulia +
+      "." +
+      " Julia is paying too little!"
+  );
+}
```
Comment (MLMeyer) [inline javascript/javascript1/week1/houseprice.js:44]: !OBS! Something is a little off with this solution 💡

You have a good approach and you are almost there 😊

**Suggestions** 
- Try to calculate the actualPrice for both Julia and Peter without the code and see if you get the same values as the result your code provides. If not, why could that be?
- Look at the variable volumeInMeters. Are you using the same values for both Julia and Peter? How could you make it so, that they have separate values for the variable volumeInMeters based on their own houses' width, height and depth? 

I hope this makes sense. Don't hesitate to reach out to me on Slack if you want to talk it through. 😊
My Slack name is Marie Louise Meyer 🙋🏼

File: javascript/javascript1/week1/EasyNames.js [hunk 4]
```diff
@@ -0,0 +1,39 @@
...
+
+console.log(
+  "The Startup: " +
+    "'" +
+    startupName +
+    "'" +
+    " contains " +
+    startupNameLength +
+    " characters."
+);
```
Comment (MLMeyer) [inline javascript/javascript1/week1/EasyNames.js:39]: Great job! 🔥

Your code is clean and well structured. 👌

My startupName became "Simple Puzzles" - now I just need to make a suitable product 😅🙌

Lastly, good attention to detail. Your string concatenation is on point. 👌 As you saw in FreeCodeCamp there are many ways to play around with inner quotes in a string, you can also use a forward slash as in the example below. 🤓

"this is a string with an inner quote \" place inner quote here \"."

File: javascript/javascript1/week1/Homework.js (hunk 1 above)
Comment (rebeccahockley) [reply javascript/javascript1/week1/Homework.js]: I think yearOfBirth should be const looking back on this - as it shouldn't change, I'll go back and amend that :D

File: javascript/javascript1/week1/dogcalculator.js (hunk 2 above)
Comment (rebeccahockley) [reply javascript/javascript1/week1/dogcalculator.js:24]: I did think this at the time about if and else statements! And in the next task, I didn't put an else condition, I should've amended - thanks for confirming this!

File: javascript/javascript1/week1/houseprice.js (hunk 3 above)
Comment (rebeccahockley) [reply javascript/javascript1/week1/houseprice.js:44]: I think I will spend some time revisiting this task as it did get a litttttle confusing there - great points made, I'll rework it

**Pull request merged**